File: 2022-2023/sourcecode/lab/lab13/DaggerHiltBasicsJava/app/src/main/java/pl/edu/uwr/pum/daggerhiltbasicsjava/data/Post.java
package pl.edu.uwr.pum.daggerhiltbasicsjava.data;

import com.google.gson.annotations.SerializedName;

public class Post {
    private int userId;
    private int id;
    private String title;

    @SerializedName("body")
    String content;

    public int getUserId() {
        return userId;
    }

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getContent() {
        return content;
}
}

File: 2022-2023/sourcecode/lab/lab13/DaggerHiltBasicsKotlin/app/src/main/java/pl/edu/uwr/pum/daggerhiltbasicskotlin/data/Post.kt
package pl.edu.uwr.pum.daggerhiltbasicskotlin.data

import com.google.gson.annotations.SerializedName

data class Post (
    val userId: Int,
    val id: Int,
    val title: String,

    @SerializedName("body")
    val content: String
)